docs/se/openapi/openapi.adoc

```diff
@@ -54,38 +53,24 @@ include::{rootdir}/includes/dependencies.adoc[]
 
 == Usage
 
-You can very simply add support for OpenAPI to your Helidon SE application. This
-document shows what changes you need to make to your application and how to access
-the OpenAPI document for your application at runtime.
+=== Automatic Registration (default)
+Simply by adding the dependency described above you add support for OpenAPI to your Helidon SE application. Because Helidon automatically discovers the OpenAPI feature, you do not have to make any changes to your application code.
 
-=== Changing your application
+=== Explicit Registration
+On the other hand, to control the behavior of the OpenAPI feature programmatically you can add and configure the OpenAPI feature explicitly as explained below.
```

Comment:
To control the behavior of the OpenAPI feature programmatically, you can add and configure the OpenAPI feature explicitly as explained below.

Reply:
Fixed